### DISCUSSION TITLE
Add `HasLivewire` Method

### DISCUSSION THREAD

**Discussion by ousid** [spatie/laravel-package-tools#51] [Ideas]
Hi, @freekmurze 

Some Packages require Laravel Livewire, as a part of the development, I thought it will be nice to have something like this:

```php
$package
      ->name('package-name')
      ->hasLivewire([
           ['some-component', SomeComponent::class],
          // ...
       ])
     //...
```
Or just we need to register them in `bootingPackage` method, since not all the packages has `Livewire` as a requirement?

If so, I can make a PR for this one :)

Thanks.

**Comment by freekmurze** [comment]
Thanks for your suggestion. I think I want to keep this package Liveware agnostic for now. I think it's easy enough to register Livewire components manually.

**Reply by ousid** [reply]
Totally agreed, thanks for the confirmation.
